_dashboards/discover/exploring-query-enhancements.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: Exploring query enhancements
+parent: Analyzing data
+nav_order: 20
+---
+
+# Exploring query enhancements
+Introduced 2.17
+{: .label .label-purple }
+
+Starting with OpenSearch 2.17, query enhancements have been made. These enhancements are experimental and may be subject to change or instability. Enhancements include the following: 
+
+- Query languages PPL and SQL, with **Query Assist** for PPL
+- Multiline query editor for PPL and SQL and autocomplete for PPL and DQL
+- Data selector with **Data Explorer** that supports index patterns, indexes, and Amazon S3 connections and data configuration that supports selecting the appropriate query language based on the data type
+- Link sharing through URLs without needing write permission to create an index pattern
+
+All existing functionality remains in OpenSearch 2.17, and the new features are designed to improve your data exploration experience. The **Enable query enhancements** is currently not on a minimum distribution. <SME: Do we need to add text about plugins?>. You can leave your feedback at <insert-link> to help the OpenSearch Project improve.
```

Comment:
@sejli Will you provide me with the link for users to leave feedback about these features? Also, please review this paragraph. Do we need to add text about plugins?

Reply:
@sejli Confirming this is on for 2.18. Do we have updates to the draft text?

Reply:
@sejli No one has provided any updates. The developers write the initial drafts for new features and the tech writer refines the content. Any updates developers can make in the PR.